src/debugger/ios/iOSPlatform.ts

```diff
@@ -98,20 +98,24 @@ export class IOSPlatform implements IAppPlatform {
                     return iosDebugModeManager.setSimulatorJSDebuggingModeSetting(/*enable=*/ true);
                 }).then(() => {
                     // Relaunch the app
-                    return this.runApp(launchArgs);
+                    return this.runApp();
                 });
             }
         });
     }
 
-    private consumeArguments(launchArgs: IRunOptions): void {
-        this.projectPath = launchArgs.projectRoot;
-        this.simulatorTarget = launchArgs.target || IOSPlatform.simulatorString;
+    private consumeArguments(): void {

```

Comment:
+1 to @joshuaskeltonm, why can't this be done in the constructor? Why do you need a separate method? I see only only call.